src/components/widgets/gcode-preview/GcodePreviewCard.vue

```diff
@@ -17,6 +17,18 @@
         </app-btn>
       </app-btn-collapse-group>
 
+      <app-btn
+        color=""
+        class="ml-1"
+        fab
+        x-small
+        text
+        :disabled="!parts.length"
```

Comment:
This didn't work for me until I changed it to this:
```suggestion
        :disabled="parts.length === 0"
```

Reply:
Actually, this is incorrect too, as parts is an object, not an array!